try:
    print(hey)
except Exception:
    pass

# print("ignored the exception")
